### DISCUSSION TITLE
Tools - Edit print parameters

### DISCUSSION THREAD

**Discussion by Nova3DBene4Mono** [sn4k3/UVtools#413] [Q&A] [answered]
Hi!
I am new here, so patience please :) I am learning to use UVtools and don't understand the settings for my 3D printer (Bene4 Mono).
I have one settings in Lychee slicer, but UVtools sets it up differently - it setts some extra time.

What to do? :)

**Comment by sn4k3** [comment]
What do you mean?
You need to post some screenshot/s and the related settings

**Reply by Nova3DBene4Mono** [reply]
The first one is from Lychee, the other is UVtools.

![Stan-003](https://user-images.githubusercontent.com/99216595/152871807-4183b727-6f69-4421-8f77-6ab366b1e3f6.PNG)
![BENE4 MONO-settings](https://user-images.githubusercontent.com/99216595/152871809-af6f4981-46de-46e7-b2b1-44306d1fe38c.PNG)

**Reply by sn4k3** [reply]
Ok, attach here a link to download the original .cws file for i check

**Comment by Nova3DBene4Mono** [comment, marked as answer]
Just a cube created in Lychee. Here is the link:
https://fromsmash.com/WtTTE38zjk-dt

**Reply by sn4k3** [reply]
I have checked, is not a bug from either side.
The thing is, the time marked at yellow you see on UVtools is the movement delay from lychee and not the "wait time before cure", which UVtools can't know which is a movement delay or a proper wait delay and set it to the "wait time before cure" at a best guess... 

So you will need to re-set that wait time on UVtools if you process the file there in order to get it correctly again after file save.

I can try to contact lychee team to see if they can adopt the UVtools schemantic of gcode in order to identify a movement delay.

**Reply by Nova3DBene4Mono** [reply]
Oh, OK, thanks a lot! 👍
So, I have to reset the UVtools settings to zero (set to Lychee settings) for each print?
Maybe, it would be good feature to add a "save setting" in UVtools for a selected printer (or for each printer).
I can´t see this option there.

**Reply by Nova3DBene4Mono** [reply]
One more thing. When I slice the model in Chitubox, save as .ctb file and then convert it to .cws in UVtools, it is OK. No "added" time.

**Reply by sn4k3** [reply]
> So, I have to reset the UVtools settings to zero (set to Lychee settings) for each print?

You have to set that "wait time before cure" on UVtools yes to fix that situation, if you process the file there.

> Maybe, it would be good feature to add a "save setting" in UVtools for a selected printer (or for each printer).
I can´t see this option there.

What do you mean?

> One more thing. When I slice the model in Chitubox, save as .ctb file and then convert it to .cws in UVtools, it is OK. No "added" time.

That's correct, because on that situation UVtools generate the correct GCode while converting formats

**Comment by Nova3DBene4Mono** [comment]
> What do you mean?"

I mean, it would be good, if we could save settings for each printer - so we do not have to reset it - UVtools would take into account the proper (saved) setting.

**Reply by sn4k3** [reply]
Do not create new comments/anwsers here. Use reply to the current topic please.

There are no printers for UVtools, it sees file formats which are printer independent.
However you can set those wait times automatic, by going to:

1. Settings -> Automations -> Light off
2. Set light off delay with extra time
3. Select your desired extra time
4. Uncheck the "Select only if value is zero"
4. Save

Now each time you open a file, it will set wait time accordingly your values

**Reply by Nova3DBene4Mono** [reply]
OK, tanks a lot!

PS: it does not help. Whatever I set, it always adds those 2,5/3,5 seconds. I use UVtools v. 2.27.7.
For me, it is OK to convert the .ctb file, but maybe this info will be helpful.

**Reply by 23-andreas** [reply]
> There are no printers for UVtools, it sees file formats which are printer independent. However you can set those wait times automatic, by going to:
> 
> 1. Settings -> Automations -> Light off
> 2. Set light off delay with extra time
> 3. Select your desired extra time
> 4. Uncheck the "Select only if value is zero"
> 5. Save

Hello, I do not see the options in Settings > Automation

Any help in understanding why is much appreciated. UVTools 3.4.1 x64 macOS

Thank you

<img width="799" alt="image" src="https://user-images.githubusercontent.com/4202141/170933122-52eb2fd1-f188-4b99-abaa-c3dc2f94fff4.png">

**Reply by sn4k3** [reply]
That option was removed since v3
Now you must use the Suggestions (Wait time before cure) as the replace-in.
Under the Shield icon near issues
